website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -208,10 +187,35 @@ There are a number of other useful configurations that can be made when initialz
 - Filter watch events using [predicates][event_filtering]
 - Choose the type of [EventHandler][event_handler_godocs] to change how a watch event will translate to reconcile requests for the reconcile loop. For operator relationships that are more complex than primary and secondary resources, the [`EnqueueRequestsFromMapFunc`][enqueue_requests_from_map_func] handler can be used to transform a watch event into an arbitrary set of reconcile requests.
 
+### Reconcile loop
 
-#### Reconcile loop
+The reconcile function is responsible for synchronizing the desired state as represented by their Specs and the actual state of the system. In this way, it works like a loop, and it does not stop until all conditionals match its implementation. 
 
-Every Controller has a Reconciler object with a `Reconcile()` method that implements the reconcile loop. The reconcile loop is passed the [`Request`][request-go-doc] argument which is a Namespace/Name key used to lookup the primary resource object, Memcached, from the cache:
+The following are a few possible return options to restart the Reconcile:
+
+- With the error:
+```go
+return ctrl.Result{}, err
+```
+
+- Without an error:
+```go
+return ctrl.Result{Requeue: true}, nil
+```
+
+- Therefore, to stop the Reconcile, use:
+```go
+return ctrl.Result{}, nil
+```
+
+- Reconcile again after X time:
+```go
+return ctrl.Result{RequeueAfter: nextRun.Sub(r.Now())}, nil
+```
+
+**Note:** For more details, check the Reconcile and its [Reconcile godoc][reconcile-godoc].
```

Comment:
Notes should be reserved for caveats or specific "something you should know" information
```suggestion
For more details, check the Reconcile and its [Reconcile godoc][reconcile-godoc].
```